### PR TITLE
feat: add `no-loader` alternative

### PR DESCRIPTION
**Pull request by nullishamy**

### CODE REVIEW

File: src/build.ts [hunk 1]
```diff
@@ -9,13 +9,24 @@ await Deno.remove(THEME_DIR, { recursive: true })
...
+
+  const { css: noLoader } = await less.render(src, vars);
+  await Deno.writeTextFile(
+    join(THEME_DIR, flavor + "-no-loader.css"),
+    noLoader
+  );
+
+  vars.globalVars.loader = true;
+  const { css: withLoader } = await less.render(src, vars);
+  await Deno.writeTextFile(join(THEME_DIR, flavor + "-loader..css"), withLoader);
```
Comment (uncenter) [inline src/build.ts]: Looks like you have an extra period here.
```suggestion
  await Deno.writeTextFile(join(THEME_DIR, flavor + "-loader.css"), withLoader);
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: src/build.ts (hunk 1 above)
Comment (uncenter) [reply src/build.ts]: Also thoughts on making the variants just `<flavor>.css` and `<flavor>-no-loader.css`? This would be a breaking change otherwise...

Comment (nullishamy) [reply src/build.ts]: Ah, didn't quite remember it would break existing installs. Let's go with that.

File: README.md [hunk 2]
```diff
@@ -36,7 +36,9 @@
 
 ## Usage
 
-1. Set the `data-theme` attribute on your Giscus `<script>` tag to `https://giscus.catppuccin.com/themes/<flavor>.css` (where `<flavor>` is one of `latte`, `frappe`, `macchiato`, or `mocha`).
+1. Set the `data-theme` attribute on your Giscus `<script>` tag to `https://giscus.catppuccin.com/themes/<flavor>-<loader>.css` (where `<flavor>` is
```
Comment (uncenter) [inline README.md]:
```suggestion
1. Set the `data-theme` attribute on your Giscus `<script>` tag to `https://giscus.catppuccin.com/themes/<flavor>.css` (where `<flavor>` is one of `latte`, `frappe`, `macchiato`, or `mocha`).

> [!TIP]
> If you don't want the Pepperjack loading spinner, add `-no-loader` after the name of the flavor: `https://giscus.catppuccin.com/themes/<flavor>-no-loader.css`.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
:(

**Pull request merged**